openedx/features/subscriptions/management/commands/send_subscriptions_expiry_emails.py

```diff
@@ -31,11 +31,20 @@ def _get_message_context(self, site):
         """
         """
         message_context = get_base_template_context(site)
-        marketing_site_root = configuration_helpers.get_value('MARKETING_SITE_ROOT')
+        marketing_urls = get_config_value_from_site_or_settings(
+            'MKTG_URLS',
+            site=site
+        )
+        marketing_site_root = marketing_urls.get('ROOT', '')
         subscriptions_marketing_url = '{marketing_root_url}subscriptions'.format(
             marketing_root_url='' if not marketing_site_root else marketing_site_root
         )
         message_context['subscriptions_marketing_url'] = subscriptions_marketing_url
+        django_settings = get_config_value_from_site_or_settings(
+            'DJANGO_SETTINGS_OVERRIDE',
+            site=site
+        )
+        message_context['ecommerce_api_url'] = django_settings['ECOMMERCE_API_URL'] if django_settings else ''
```

Comment:
@taimoor-ahmed-1 do we add the `ECOMMERCE_API_URL` in site configuration?

Reply:
yes, `ECOMMERCE_API_URL` is in `ALLOWED_DJANGO_SETTINGS_OVERRIDE` list and it will always be present in `DJANGO_SETTINGS_OVERRIDE` object in site configurations.
<img width="1364" alt="Screenshot 2021-08-11 at 12 02 17 PM" src="https://user-images.githubusercontent.com/68893403/128984263-713f9001-b47a-4295-89f9-1924560301ab.png">